scripts/helpers/utilities.mk

```diff
@@ -394,7 +397,7 @@ define prepare_loader_cache
     if [[ -n "$(TARGET64)" ]]; then \
         > "$(DYNAMIC_LOAD)/at64.ld.conf"; \
         if [[ $1 -eq 1 ]]; then \
-            echo "$(AT_DEST)/syslib-override"   >  "$(DYNAMIC_LOAD)/at64.ld.conf"; \
+            echo "$(AT_DEST)/lib64/glibc-hwcaps"   >  "$(DYNAMIC_LOAD)/at64.ld.conf"; \
```

Comment:
OK. This is replacing the syslib-override mechanism with glibc-hwcaps for all AT versions supported at the moment.